docker-compose-dev.yml

```diff
@@ -0,0 +1,56 @@
+# dev compose
+# extend from docker-compose-common.yml
+geocoder:
+  extends:
+    file: docker-compose-common.yml
+    service: geocoder
+  links:
```

Comment:
Can the links be moved into the "common" file when they're the same in both dev and prod?